### PR TITLE
Add tracing section

### PR DESCRIPTION
**Pull request by sponomarev**

### CODE REVIEW

File: docs/go_tracing.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# AnyCable-Go Instrumentation
+
+AnyCable-Go assigns a random unique `sid` or use a provided one from `X-Request-ID` HTTP header
```
Comment (palkan) [inline docs/go_tracing.md]:
```suggestion
AnyCable-Go assigns a random unique `sid` (_session ID_) or use the one provided in the `X-Request-ID` HTTP header
```

File: docs/go_tracing.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+to each websocket connection and passes it with requests to RPC service. This identificator is also
+available in logs and you can use it to trace a request's pathway throught the whole LB->WS->RPC stack.
```
Comment (palkan) [inline docs/go_tracing.md]:
```suggestion
available in logs and you can use it to trace a request's pathway through the whole Load Balancer -> WS Server -> RPC stack.
```

File: docs/go_tracing.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Logs example:
+
+```
+D 2019-04-25T18:41:07.172Z context=node sid=FQQS_IltswlTJK60ncf9Cm Incoming message: &{subscribe {"channel":"PresenceChannel"} }
+D 2019-04-25T18:41:08.074Z context=pubsub Incoming pubsub message from Redis: {"stream":"presence:Z2lkOi8vbWFuYWdlYmFjL1NjaG9vbC8xMDAwMjI3Mw","data":"{\"type\":\"presence\",\"event\":\"user-presence-changed\",\"user_id\":1,\"status\":\"online\"}"}
+```
+
+## Using with Heroku
+
+Heroku assigns `X-Request-ID` [automatically on router level](https://devcenter.heroku.com/articles/http-request-id).
```
Comment (palkan) [inline docs/go_tracing.md]:
```suggestion
Heroku assigns `X-Request-ID` [automatically at the router level](https://devcenter.heroku.com/articles/http-request-id).
```

File: docs/logging.md [hunk 4]
```diff
@@ -17,6 +17,8 @@ You can also specify your own logger instance for full control:
 AnyCable.logger = MyLogger.new
 ```
 
+If you use Anycable with Rails, see corresponding section in [Using with Rails](using_with_rails#logging)
```
Comment (palkan) [inline docs/logging.md]:
```suggestion
If you use AnyCable with Rails see the corresponding section in [Using with Rails](using_with_rails#logging)
```

File: docs/using_with_rails.md [hunk 5]
```diff
@@ -114,6 +114,17 @@ config.log_level = :debug
 Rails.logger.level = :debug if AnyCable.config.debug?
 ```
 
+### Log tracing
+
+Using with Rails, AnyCable adds session id tag `sid` to each log entry produced during RPC message handling. You can use it to trace a request's pathway throught the whole LB->WS->RPC stack.
```
Comment (palkan) [inline docs/using_with_rails.md]:
```suggestion
Using with Rails, AnyCable adds a _session ID_ tag (`sid`) to each log entry produced during the RPC message handling. You can use it to trace the request's pathway throught the whole Load Balancer -> WS Server -> RPC stack.
```

**Review by palkan — APPROVED** [review]
👍 

Added some language suggestions 😉

**Comment by sponomarev** [comment]
I don't agree with all the changes  🤷‍♂ , but anyway 👍

**Review by palkan — APPROVED** [review]

**Pull request merged**